.github/ISSUE_TEMPLATE/add-new-feature---behavior.md

```
@@ -20,22 +20,26 @@ The current serverless-appsync-plugin does not support RDS as a datasource.
 A clear and concise description of what the current behavior is.
 
 **Example:**
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)